build.rs

```diff
@@ -8,4 +8,8 @@ fn main() {
     if rustc::is_min_version("1.13.0").map(|(is_min, _)| is_min).unwrap_or(true) {
         println!("cargo:rustc-cfg=__unicase__default_hasher");
     }
+
+    if rustc::is_min_version("1.31.0").map(|(is_min, _)| is_min).unwrap_or(true) {
```

Comment:
We should probably use `.unwrap_or(false)` since if we can't verify the version then we probably don't want to assume these features are supported.

Reply:
It's been so long, I can't remember. Something that comes to mind is if nightly were to output an odd version string. Either way, I'm not too concern with it.